test/NuGet.Core.Tests/NuGet.Protocol.Core.v3.Tests/FeedTypeUtilityTests.cs

```diff
@@ -46,6 +46,19 @@ public void FeedTypeUtility_VerifyBadSourceIsUnknown()
             Assert.Equal(FeedType.FileSystemUnknown, type);
         }
 
+        [Theory]
+        [InlineData("../foo/packages")]
+        [InlineData(@"..\foo\packages")]
+        [InlineData(@"packages")]
+        public void FeedTypeUtility_VerifyRelativePathIsUnknown(string source)
+        {
+            // Arrange & Act
+            var type = FeedTypeUtility.GetFeedType(new PackageSource(source));
+
+            // Assert
+            Assert.Equal(FeedType.FileSystemUnknown, type);
```

Comment:
I suggest double checking that FileSystemUnknown will be handled for each resource type, otherwise there could be null refs if this actually gets hit.